incubator/kube2iam-umbrella/values.yaml

```diff
@@ -0,0 +1,43 @@
+# Default values for kube2iam-umbrella.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+define:
+  - &IAM_BASE_ROLE_ARN ""  # Role base ARN. Ex: arn:aws:iam::312041674177:role/
+  - &IAM_DEFAULT_ROLE ""   # Default role
+
+kube2iam:
+  extraArgs:
+    base-role-arn: *IAM_BASE_ROLE_ARN
+    default-role: *IAM_DEFAULT_ROLE
+  #   api-server: ...
+  #   api-token: ...
+
+  host:
+    ip: $(HOST_IP)
```

Comment:
This is just a string that makes to use ENV var HOST_IP that is specified by K8S